package.json

```diff
@@ -33,11 +33,12 @@
     "postinstall": "echo 'root postinstall' && npm run link && npm run all install && npm run build",
     "clean-deps": "npm run all clean-deps && rm -rf node_modules",
     "docker": "./scripts/run-docker-local.sh",
-    "lint-js": "eslint --fix scripts/*.js packages/ts/*.js cli/*.js cli/**/*.js",
-    "lint-changed": "git diff --name-only | grep '\\.js$' | xargs npx eslint --fix",
-    "lint-coffee": "coffeelint scripts/**/*.coffee",
-    "lint": "npm run lint-js && npm run lint-coffee",
-    "pretest": "npm run lint && npm run all lint && npm run test-scripts",
+    "lint-js": "eslint --fix .",
+    "lint-ts": "eslint --fix . --ext .ts",
+    "lint-changed": "git diff HEAD --name-only | grep '\\.\\(ts\\|js\\)$' | xargs npx eslint --fix ''",
```

Comment:
before, this was only linting `unstaged` files. using `git diff HEAD` includes both staged & unstaged